LANGUAGES/languages.json

```diff
@@ -360,51 +360,51 @@
         "name":"Turkish (Türkçe)",
         "FontFamily":"Courier New",
         "FontSize":16,
-        "Author":"Erhan Kültür/crasadure",
+        "Author":"Erhan Kültür/crasadure, Oğuz Ersen",
         
         "STR_ERROR":"Hata",
         "STR_WARNING":"Uyarı",
         "STR_INFO":"Bilgi",
         "STR_INCORRECT_DIR":"Lütfen doğru dizin altında çalıştırın!",
-        "STR_INCORRECT_TREE_DIR":"Dosyayı burada çalıştırma, lütfen yayınlanan en son kurulum paketini indirin ve oradan çalıştırın.",
+        "STR_INCORRECT_TREE_DIR":"Dosyayı burada çalıştırmayın, lütfen yayınlanan en son kurulum paketini indirin ve oradan çalıştırın.",
```

Comment:
Virgülden sonraki kısımda çoğul şahıs kullanıldığı için tutarlı olması için değiştirildi.